File: Data Structures & Algorithms/reverse-string/submission-0.java
class Solution {
    public void reverseString(char[] s) {
        int i = 0, r = s.length-1;
        while(i < r){
            char c = s[i];
            s[i++] = s[r];
            s[r--] = c;
        }
    }
}

File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = [0] * 26
        if(len(s) != len(t)):
            return False

        for i in range(len(s)):
            idx = ord(s[i]) - ord('a')
            idx2 = ord(t[i]) - ord('a')
            count[idx] = count[idx] + 1
            count[idx2] = count[idx2] - 1

        for i in count:
            if i!=0:
                return False

        return True
        
        
       
            